client/src/Utils/Validations/Validator.ts

```diff
@@ -4,8 +4,13 @@ const phoneValidation = (phoneNumber: string) => {
     return IsraelPhoneNumberRegEx.test(phoneNumber);
 }
 
+const formValidation = (form: any): boolean => {
+    return Object.values(form).some((value: any) => value && value.isValid !== undefined && !value.isValid)
```

Comment:
condition can just be value.isValid===false

Reply:
sometimes fields are null so not exactly, but it can be value && value.isValid===false

Reply:
yup